api/views/event.py

```diff
@@ -0,0 +1,21 @@
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework import viewsets, mixins
+from rest_framework.permissions import IsAuthenticated
+
+from api.serializers import EventSerializer
+from canvas.models import Event
+
+
+class EventViewSet(mixins.CreateModelMixin, mixins.DestroyModelMixin, mixins.UpdateModelMixin,
```

Comment:
You can use compact version of viewsets.ModelViewset

Reply:
I wanted to disable the list() method specifically since there's currently no need to retrieve all events in the system and allowing that to be possible has its own permission issues